server/controllers/goalsController.test.ts

```diff
@@ -6,67 +8,73 @@ import { auditServiceMock } from '../../tests/mocks/auditServiceMock'
 import WorkAndSkillsPageService from '../services/workAndSkillsPageService'
 import GoalsController from './goalsController'
 
-let req: any
-let res: any
-let controller: GoalsController
+describe('Prisoner goals controller', () => {
```

Comment:
I had to 'touch' this test class (because of the changes to the `workAndSkillsPageService`), so ended up refactoring it so that it was more like what we've done in PLP and CIAG wrt the patterns for setting up the mocks etc